Procfile

```diff
@@ -0,0 +1 @@
+web: bin/boot
```

Comment:
It's from [heroku-buildpack-static](https://github.com/hone/heroku-buildpack-static/blob/master/scripts/boot).